src/platform/testing/e2e/megamenu/megamenu.js

```diff
@@ -277,7 +277,7 @@ function testDataDrivenMegamenu(client, path) {
     'css selector',
     '#vetnav-column-one-col li.mm-link-container',
     results => {
-      client.expect(results.value.length).to.equal(8);
+      client.expect(results.value.length).to.equal(7);
```

Comment:
Okay, I didn't change anything about this particular section, yet this still passes... 🤷

Reply:
But the test does fail without this change when run with the Selenium server and chromedriver updates involved with the GHA migration. And this assertion matches what you'd see in prod. So we still need this change.